api/build.gradle

```diff
@@ -476,7 +476,7 @@ dependencies {
     providedCompile group: 'javax.servlet', name: 'javax.servlet-api', version: '3.1.0'
 
     __tools__Compile 'commons-cli:commons-cli:1.5.0'
-    __tools__Compile 'com.opencsv:opencsv:4.6'
+    __tools__Compile 'com.opencsv:opencsv:5.6'
```

Comment:
we must have removed the previous tools which used this ... might as well bump the version when we bring it back